packages/cactus-test-cmd-api-server/package.json

```diff
@@ -69,7 +69,8 @@
     "@hyperledger/cactus-core-api": "0.7.0",
     "@hyperledger/cactus-plugin-keychain-vault": "0.7.0",
     "express-jwt-authz": "2.4.1",
-    "jose": "1.28.1"
+    "jose": "1.28.1",
+    "artillery": "1.7.2"
```

Comment:
@travis-payne The artillery benchmark is in the other package (not the test one but the cmd-api-server package itself).
Please either move this devDependency over there or bring the test case over here just so that they are in the same package.

`/packages/cactus-cmd-api-server/src/test/typescript/benchmark/artillery-api-benchmark.test.ts`